examples/notebooks/persistent_dataset_speed.ipynb

```diff
@@ -139,12 +138,10 @@
     "                    best_metric_epoch = epoch + 1\n",
     "                    torch.save(model.state_dict(), 'best_metric_model.pth')\n",
     "                    print('saved new best metric model')\n",
-    "                print('current epoch: {} current mean dice: {:.4f} best mean dice: {:.4f} at epoch {}'.format(\n",
-    "                    epoch + 1, metric, best_metric, best_metric_epoch))\n",
-    "        print('time consuming of epoch {} is: {:.4f}'.format(epoch + 1, time.time() - epoch_start))\n",
+    "                print(f\"current epoch: {epoch + 1} current mean dice: {metric} best mean dice: {best_metric} at epoch {best_metric_epoch}\")\n",
```

Comment:
The previous string only keeps 4 places after the decimal point.
Something like: `f"XXX{data: 4}"`

Thanks.